LoadTests/src/test/scala/default/RecordingSimulation.scala

```diff
@@ -65,7 +70,10 @@ class RecordingSimulation extends Simulation {
 					.headers(headers_1),
 				http("Load crest")
 					.get("/public/govuk_template/assets/stylesheets/images/govuk-crest-2x.png")
-					.headers(headers_1)))
+					.headers(headers_1),
+				http("Load accessible autocomplete")
+  				.get("/assets/stylesheets/accessible-autocomplete.min.css")
+  				.headers(headers_6)))
```

Comment:
We probably don't need this bit any more (since the CSS file is now included in the main SCSS file)

Reply:
Good point - I meant to remove this, thanks!